src/components/navigation-header/index.tsx

```diff
@@ -1,9 +1,37 @@
+import Link from 'next/link'
 import HashiCorpLogo from '@hashicorp/mktg-logos/corporate/hashicorp/logomark/white.svg?include'
-import HeaderSearchInput from 'components/header-search-input'
 import InlineSvg from '@hashicorp/react-inline-svg'
+import HeaderSearchInput from 'components/header-search-input'
 import ProductChooser from 'components/product-chooser'
+// TODO: we'll need a programatic way to get this data when there are more products
+import waypointData from 'data/waypoint.json'
 import s from './style.module.css'
 
+/**
+ * This is a temporary component. Once all of the nav links point to internal paths,
+ * it won't be needed anymore. Not planning to move into a separate file.
+ */
+const NavLink: React.FC<{
```

Comment:
nit: `React.FC` implicitly makes `children` a valid prop in the component's type definition, so if we don't do anything with `children`, it might not be what we want. I see this is a temporary component though, so not a big deal.

ref: https://www.harrymt.com/blog/2020/05/20/react-typescript-react-fc.html

Reply:
Didn't think about that! Very good point. Updated. 😊